client/src/thin_client.rs

```diff
@@ -233,7 +234,7 @@ impl ThinClient {
                     // all pending confirmations. Resending the transaction could result into
                     // extra transaction fees
                     pending_confirmations -= confirmed_blocks;
-                    wait_time = MAX_PROCESSING_AGE * pending_confirmations;
+                    wait_time = cmp::max(wait_time, MAX_PROCESSING_AGE * pending_confirmations);
```

Comment:
prefer `wait_time = wait_time.max(MAX_PROCESSING_AGE * pending_confirmations);` ?

Reply:
Yes, that looks better. I was wondering if there's a better way of doing this.. thanks!

Reply:
saves a `use`, dunno if it's more canonical

Reply:
I don't like the way we do `use`, would prefer we did it more localized